src/main/frontend/handler/page.cljs

```diff
@@ -464,6 +464,11 @@
         (when (and file (not journal?))
           (rename-file! file new-file-name-body (fn [] nil)))
 
+
+        (let [home (get (state/get-config) :default-home {})]
+          (when (= old-page-name (string/lower-case (get home :page "")))
```

Comment:
Better to use `util/page-name-sanity-lc` instead of `string/lower-case` for error-proof